lib/defaults.js

```diff
@@ -0,0 +1,60 @@
+var defaults = {
+  jquery: {
+    style: 'jquery',
+    noSemver: false,
```

Comment:
Pretty much all the options that have the same default for both styles shouldn't be options at all.